sphinx_click/ext.py

```diff
@@ -51,8 +51,11 @@ def _write_opts(opts):
     help = opt.help or ''
     extra = []
     if opt.default is not None and opt.show_default:
-        extra.append('default: %s' %
-                     (', '.join('%s' % d for d in opt.default) if isinstance(
+        if isinstance(opt.show_default, str):
```

Comment:
It'd be good to add comment here stating that this is a click 7 only thing, though click 6 is pretty long in the tooth now so I'm not sure it matters much.

Reply:
:+1: